web/src/components/FilingsCalendar.tsx

```diff
@@ -24,7 +25,12 @@ export const FilingsCalendar = (props: Props): ReactElement => {
   const Config = getMergedConfig();
   const { userData, update } = useUserData();
   const isLargeScreen = useMediaQuery(MediaQueries.tabletAndUp);
-  const taxFilings = userData?.taxFilingData.filings || [];
+  const taxFilings =
+    userData?.taxFilingData.filings.sort((a, b) =>
+      parseDateWithFormat(a.dueDate, "YYYY-MM-DD").isBefore(parseDateWithFormat(b.dueDate, "YYYY-MM-DD"))
```

Comment:
a common comment I'm leaving, but I think this one-liner would be more readable if you extracted some of the logic into a well-named function. For example:

```
const sortFilingsEarliestToLatest = () => {...} // is this even right? custom sorts are hard to read. hence my desire for descriptive names so that the code can be understood
const orderedTaxFilings = userData?.taxFilingData.filings ? sortFilingsEarliestToLatest(userData.taxFilingData.filings) : []
```